ewardrobe/ewardrobe_app/queries/orders.py

```diff
@@ -0,0 +1,31 @@
+from ewardrobe_app.models import Basket
+from django.contrib.auth.models import User
+from ewardrobe_app.models import STATUS_CHOICES
+
+
+TRANSITIONS = {
```

Comment:
I have a feeling that it should be possible to take it from `django_fsm`

Reply:
done